agent/consul/acl_replication_test.go

```diff
@@ -126,6 +126,14 @@ func TestACLReplication_diffACLPolicies(t *testing.T) {
 
 func TestACLReplication_diffACLTokens(t *testing.T) {
 	local := structs.ACLTokens{
+		// just after a 1.3->1.4 upgrade this will happen in the remote
```

Comment:
"in the reomte" is confusing since this is the local tokens. You mean this is modelling the secondary DC state right?